src/model/attendee.py

```diff
@@ -87,22 +91,25 @@ def add_attendee(data):
         session.close()
         return didSucceed
 
+
 def get_attendee(email):
     try:
         session = Session()
-        attendee = session.query(Attendee).filter_by(email = email).first()
+        attendee = session.query(Attendee).filter_by(email=email).first()
         return attendee
     except Exception as e:
         print(e)
 
+
 def get_attendee_by_id(id):
-    attendee = session.query(Attendee).filter_by(id = id).first()
+    attendee = session.query(Attendee).filter_by(id=id).first()
     if attendee is None:
         return "Attendee not found", 404
     else:
         result = attendee_schema.dump(attendee)
         return result, 200
 
+
 def decode_auth_token(token):
```

Comment:
JWT encode and decode methods are really starting to look out of place in `attendee.py`, we should shift them to somewhere else like a `jwt.py` or something. Not sure how Python people handle their utility/helper stuff... that's up to you I guess